extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -667,6 +704,39 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     }
   }
 
+  /** @override */
+  setupListenersForFluid(iframe) {
+    listenFor(iframe, 'creative_geometry_update', data => {
+      // The first creative_geometry_update message will contain bad
+      // geometric data, as it will have been computed using the initial,
+      // incorrect, iframe style. We use this first message as a
+      // triggering point to restyle the iframe, which will in turn cause
+      // safeframe to send another creative_geometry_update message, this
+      // time containing the correct information.
+      const styleString = iframe.getAttribute('style');
+      if (/width: 0px/.test(styleString) &&
+          /height: 0px/.test(styleString)) {
```

Comment:
What about 0em, 0vw, just plain 0, etc.? Do we need to test for those?
What about width:0px (no space after colon), etc.?

Reply:
I will certainly add a test to ensure that the iframe is initialized with style 'width: 0px, height: 0px'. This is a style applied _somewhere_ by AMP, so I don't think there's a risk of any other units appearing besides pixels.

Reply:
Ok, that makes sense.